src/main/resources/templates/edit_user.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeLeaf.org">
+<head>
+    <meta charset="ISO-8859-1">
+    <title>Edit user</title>
+    <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}">
+</head>
+<body>
+<div align="center">
+    <br/>
+    <form action="#" th:action="@{/save}" th:object="${user}" method="post">
+        <table class="table_1">
+            <caption>Edit user</caption>
+            <tr>
+                <td>Id</td>
+                <td><input type="text" th:field="*{id}" readonly="readonly"/></td>
+            </tr>
+            <tr>
+                <td>Name</td>
+                <td><input type="text" th:field="*{name}"/></td>
+            </tr>
+            <tr>
+                <td>Age</td>
+                <td><input type="number" th:field="*{age}"/></td>
+            </tr>
+            <tr>
+                <td>Email</td>
+                <td><input type="text" th:field="*{email}"/></td>
+            </tr>
+            <tr>
+                <td>Birthday</td>
+                <td><input type="text" th:field="*{birthday}"/></td>
```

Comment:
Why not date type?
https://stackoverflow.com/questions/38693971/input-type-date-thymeleaf